VirtualPet1/src/index.ts

```diff
@@ -0,0 +1,43 @@
+class VirtualPet {
+    name: string;
+    hunger: number;
+    happiness: number;
+  
+    constructor(name: string) {
+      this.name = name;
+      this.hunger = 50; 
+      this.happiness = 50; 
+    }
```

Comment:
Instead of hardcoding 50, it should be a default value for the constructor
```suggestion
    constructor(name: string, hunger: number = 50, happiness: number = 50) {
        this.name = name
        this.hunger = hunger
        this.happiness = happiness
    }
```